composer.json

```diff
@@ -208,7 +208,7 @@
         "behat/mink": "^1.7",
         "behat/mink-extension": "^2.2",
         "behat/mink-goutte-driver": "^1.2",
-        "behat/mink-selenium2-driver": "1.3.x-dev",
+        "behat/mink-selenium2-driver": "1.3.x-dev || dev-master",
```

Comment:
Whats the motivation for this change? If anything, You could try removing the requirement from composer.json and just letting dependencies handle it.

Reply:
The version installed and locked in composer.json was not in sync with composer.json version, so I had to add this in composer.json file. Nothing was installed and updated related to behat/mink-selenium2-driver package